docs/content_management/configure_ct_field_settings.md

```diff
@@ -4,14 +4,21 @@ description: Configure Field settings in Content Types.
 
 # Configure Content Type Fields
 
-When you [modify a Content Type](create_edit_content_types.md) and add or modify [Fields](content_model.md#fields-and-field-types), 
+When you work with a Content Type to add or modify [Fields](content_model.md#fields-and-field-types), 
 you must configure different settings that control the way [[= product_name =]] 
-treats the contents of each Field. Apart from general settings, certain Field Types, for example, [Page](#default-configuration-of-pages) and [Relation](#content-relation-settings) Field Types may have additional settings.
+treats the contents of each Field. Apart from general settings, certain Field Types, for example, [Landing Page](#default-configuration-of-pages) and [Content relation](#content-relation-settings) Field Types may have additional settings.
 
 You can only create or modify Content Types when your [user Role](../permission_management/work_with_permissions.md) has the `ContentType/Create` or `ContentType/Update` permission.
 
-To modify Field settings, in the **Field definitions** section, expand the **Landing Page** header. 
-When done, click **Save and close** to apply changes to the Content Type.
+1. [Open a Content Type](create_edit_content_types.md) for editing.
```

Comment:
Link to Content Type added above, in my opinion doesn't fix here, in the procedure.
```suggestion
To modify field settings:

1\. In the left panel, go to **Content** -> **Content Types**.

2\. Open Content Type group, from the list choose the Content Type and click **Edit**.

3\. Go to **Field definitions** section. Here you can modify already added Fields or add new ones from the **Field types** toolbar on the right side. To modify Field settings, click an arrow icon to expand the window and then implement the changes.

4\. You can save your changes in two ways:

- click **Save** to save your changes and stay in edit mode.

- click **Save and close** to save your changes and exit edit mode.

5\. To discard your changes:

- click **Discard** if you are creating a new Content Type.

- click **Delete draft** if you are editing an existing Content Type.
```

Reply:
The link in the fiirst paragraph leads to a concept. Here we introduce the procedure: "How to open a CT for editing". And it makes a good first step. Users who know how to get there will skip the link.